### PR TITLE
Enterprise doc updates 

### PR DESCRIPTION
**Pull request by caitmuenster**
This PR: 

- creates a new page describing how developers can add enterprise policy support to their extensions
- updates the Enterprise overview page 
- updates the sidebar 

@mkaply, could you take a look at this?

### CODE REVIEW

**Comment by mkaply** [comment]
Took a quick look and it looks good to me.

File: src/content/documentation/enterprise/adding-policy-support-to-your-extension.md [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+
+For local group testing, you can place the ADMX file in `C:\Windows\PolicyDefinitions` and the ADML file in `C:\Windows\PolicyDefinitions\en-US`. The files must have the same basename. You can test the policy by running "Edit Group Policy" (GPEDIT.MSC) on Windows Professional and navigating to Administrative Templates->Mozilla under Computer or User Configuration.
+
+Below are two basic examples: 
+
+#### ADMX file
+
+<!-- START: Syntax Highlighting -->
+```xml
+
```
Comment (muffinresearch) [inline src/content/documentation/enterprise/adding-policy-support-to-your-extension.md:112]: Presumably this empty line could be removed?

File: src/content/documentation/enterprise/adding-policy-support-to-your-extension.md [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+    </policy>
+  </policies>
+</policyDefinitions>
+```
+<!-- END: Syntax Highlighting -->
+
+#### ADML file 
+<!-- START: Syntax Highlighting -->
+```xml
+
```
Comment (muffinresearch) [inline src/content/documentation/enterprise/adding-policy-support-to-your-extension.md:172]: Presumably this empty line could be removed?

File: src/content/documentation/enterprise/adding-policy-support-to-your-extension.md [hunk 3]
```diff
@@ -0,0 +1,255 @@
...
+<!-- END: Syntax Highlighting -->
+
+If you want to learn more about ADMX/ADML files, you can look at the templates we've built for Firefox [here](https://github.com/mozilla/policy-templates/tree/master/windows).
+
+### macOS
+macOS uses configuration profiles for policy configuration. macOS administrators will need to combine the changes needed for your extension with their existing Firefox configuration profile. An example plist file is shown below.
+
+<!-- START: Syntax Highlighting -->
+```xml
+
```
Comment (muffinresearch) [inline src/content/documentation/enterprise/adding-policy-support-to-your-extension.md:215]: Presumably this empty line could be removed?

File: src/content/documentation/enterprise/adding-policy-support-to-your-extension.md [hunk 4]
```diff
@@ -0,0 +1,255 @@
...
+        </dict>
+      </dict>
+    </dict>
+  </dict>
+</plist>
+```
+<!-- END: Syntax Highlighting -->
+
+### Linux 
+For Linux, you can provide a sample policies.json similar to what was provided earlier in this document. Policies.json is the only mechanism supported on Linux.
```
Comment (muffinresearch) [inline src/content/documentation/enterprise/adding-policy-support-to-your-extension.md:245]: policies.json could be in backticks.

**Review by muffinresearch — APPROVED** [review]
r+wc

**Comment by caitmuenster** [comment]
Thanks @muffinresearch! Just made the suggested changes.

**Pull request merged**